parts/linux/cloud-init/artifacts/components.json

```diff
@@ -658,7 +658,7 @@
           }
         },
         "azurelinux": {
-          "current": {
+          "v3.0": {
```

Comment:
Bear with me, I should have created another branch to modify this. Now I modified the schema to support `v3.0` for explicit Azure Linux v3.